tutorial/tuto5.py

```diff
@@ -28,8 +28,8 @@ def improved_table(self, header, data):
         # Column widths
         w = [40, 35, 40, 45]
         # Header
-        for i in range(0, len(header)):
-            self.cell(w[i], 7, header[i], 1, 0, "C")
+        for w_, header_ in zip(w, header):
+            self.cell(w_, 7, header_, 1, 0, "C")
```

Comment:
`width` and `header_text` or `header_str` would be more meaningful variable names here